source/gcp/prerequisites/credentials.rst

```diff
@@ -9,46 +9,81 @@
 Configuring GCP credentials
 ===========================
 
-In order to make the Wazuh GCP module pull log data from Google Pub/Sub, it will be necessary to provide access credentials so it can connect to them.
+In order to make the Wazuh GCP module pull log data from Google Pub/Sub or Google Storage, it will be necessary to provide access credentials so it can connect to them.
 
-To do this, we will need to create a service account with the Pub/Sub permissions and then create a key. It is important to save this key as a JSON file. We will use this file as credentials for the GCP module.
+To do this, it is recommended to create a service account with the Pub/Sub or Storage permissions and then create a key. It is important to save this key as a JSON file as this file will be used as the authentication method for the GCP module.
```

Comment:
```suggestion
To do this, it is recommended to create a service account with the Pub/Sub or Storage permissions and then create a key. It is important to save this key as a JSON file as it will be used as the authentication method for the GCP module.
```